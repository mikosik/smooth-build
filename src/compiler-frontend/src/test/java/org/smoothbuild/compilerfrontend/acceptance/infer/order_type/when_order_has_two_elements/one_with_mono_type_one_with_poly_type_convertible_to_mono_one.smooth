Int myIntId(Int i) = i;
A myId(A a) = a;
result = [myIntId, myId];
