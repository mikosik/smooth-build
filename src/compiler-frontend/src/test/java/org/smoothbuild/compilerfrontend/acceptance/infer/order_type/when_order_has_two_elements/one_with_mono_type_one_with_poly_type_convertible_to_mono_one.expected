Int myIntId<>(Int i)
  = i;
A myId<A>(A a)
  = a;
[(Int)->Int] result<>
  = [myIntId<>, myId<Int>];
